.github/workflows/main.yaml

```diff
@@ -30,10 +30,17 @@ jobs:
           restore-keys: |
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install python dependencies
+        run: python -m pip install pipenv
+      - run: pipenv sync
       - name: Install dependencies
-        run: yarn install
+        run: pipenv run yarn install
       - name: Execute lint
-        run: yarn lint
-      - run: yarn build
+        run: pipenv run yarn lint
+      - run: pipenv run yarn build
```

Comment:
I think we don't need to prefix lint and build with `pipenv`

Reply:
ok, checking and removing